packages/strapi-admin/ee/admin/containers/Roles/CreatePage/ConditionsModal/ConditionsSelect/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Select from 'react-select';
+import { useIntl } from 'react-intl';
+import MenuList from './MenuList';
+import ClearIndicator from './ClearIndicator';
+import DropdownIndicator from './CustomDropdownIndicator';
+import IndicatorSeparator from './IndicatorSeparator';
+import SingleValue from './SingleValue';
+import selectStyle from '../utils/selectStyle';
+
+const Wrapper = styled.div`
+  padding-left: 30px;
+  width: 60%;
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'default')};
+`;
+
+const ConditionsSelect = ({
+  arrayOfOptionsGroupedByCategory,
+  name,
+  onCategoryChange,
+  onChange,
+  value,
+}) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Wrapper
+      // TODO
+      disabled={false}
+    >
+      <Select
+        components={{
+          ClearIndicator,
+          DropdownIndicator,
+          IndicatorSeparator,
+          SingleValue,
+          MenuList,
+        }}
+        arrayOfOptionsGroupedByCategory={arrayOfOptionsGroupedByCategory}
+        // TODO
+        isDisabled={false}
+        name={name}
+        onChange={onChange}
+        isClearable={false}
+        isLoading={false}
+        closeMenuOnSelect={false}
+        isSearchable={false}
+        hideSelectedOptions={false}
+        placeholder={formatMessage({ id: 'Settings.permissions.conditions.anytime' })}
+        onCategoryChange={onCategoryChange}
+        options={[]}
+        styles={selectStyle}
```

Comment:
Agree! 
However, since @mfrachet put this in a utility in the strapi-helper-plugin, I'd rather wait for his PR to be merged and then I'll apply the modification, is ok for you?